src/Common/src/System/Drawing/ColorUtil.netcoreapp20.cs

```diff
@@ -0,0 +1,98 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Reflection;
+
+namespace System.Drawing
+{
+    // System.Drawing.Common uses several members on System.Drawing.Common which are implemented in .NET Core 2.0, but
+    // not exposed. This is a helper class which allows System.Drawing.Common to access those members through reflection.
+    internal static class ColorUtil
+    {
+        private const short StateKnownColorValid = 0x0001;
+        private const short StateNameValid = 0x0008;
+        private const long NotDefinedValue = 0;
+
+        private static readonly ConstructorInfo s_ctorKnownColor; // internal Color(KnownColor knownColor)
+        private static readonly ConstructorInfo s_ctorAllValues; // private Color(long value, short state, string name, KnownColor knownColor)
+        private static readonly FieldInfo s_fieldKnownColor; // private readonly short knownColor
+        private static readonly FieldInfo s_fieldState; // private readonly short state
+
+        static ColorUtil()
```

Comment:
Do we think reflection is the best option here? Why not just source sharing and changing the visibility?